libs/portals/my-pages/core/src/hooks/useScrollPosition/useScrollPosition.ts

```
@@ -0,0 +1,80 @@
+  currPos: {
+    x: number
+    y: number
+  }
+}
+
+interface GetScrollPositionProps {
+  element?: MutableRefObject<HTMLElement | null>
+  useWindow?: boolean
+}
+
+const isBrowser = typeof window !== `undefined`
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)